### ISSUE TITLE
Update the Family Register page

### ISSUE THREAD

**Issue by msschroeder** [OpenSRP/opensrp-client-chw#1316]
The client would like to add the date the family was created. 

- [ ] Add the {event_date} to the Family profile (pulling form the Family Details form). It would be something like:

[Surname] Family
{fam_village} [interpunct] Created: {event_date}

![Screenshot_20200907-150830](https://user-images.githubusercontent.com/20777928/92386459-81f64480-f11c-11ea-9f44-3917564bd6d3.png)

**Comment by msschroeder** [comment]
@rowo Is this okay with you?

**Comment by ellykits** [comment]
@msschroeder Technically we will need to do the following for this issue considering @rowo is okay with the suggestion:
- Add a new column (**dateCreated**) on the **family** table. The date created should be obtained from events. For newly registered families this column will be populated but for the already registered family refer to the next step
- Write a migration script to populate newly added column with the  dates the families were created. This will be done on the Android client.
- Finally update the family profile user interface to include the new display field (this should be made configurable)

This would take approximately **1** day to complete the above tasks.

**Comment by rowo** [comment]
@msschroeder @ellykits yes that area is free for all whatever kind of meta information is useful for the user
